store/actions/AuthActions.js

```diff
@@ -53,6 +54,7 @@ export const loginHandler = (credential, rememberMe) => {
       .then(async (response) => {
         dispatch(setLoading(false));
         dispatch(login(response.data, rememberMe));
+        dispatch(setStatus(response.data.id, response.data.token, { "naloxoneAvailability": response.data.naloxoneAvailability }))
```

Comment:
Is this supposed to be setting the online status instead?

Reply:
Online status is already set by the backend through the endpoint

Reply:
Oh i see, but why do we need to update the naloxone at login? The user hasnt touched the switch yet.

Reply:
oh is it because its from the server and you are adding it to redux state lmao im stupid

Reply:
I was about to say that, glad you figured~